client/me/purchases/purchases-list/index.jsx

```diff
@@ -38,6 +46,36 @@ class PurchasesList extends Component {
 		return ! this.props.sites.length;
 	}
 
+	renderConciergeBanner() {
+		const { nextAppointment, scheduleId, hasAvailableConciergeSessions } = this.props;
+
+		if ( null === hasAvailableConciergeSessions ) {
+			return <ConciergeBanner bannerType="placeholder" />;
+		}
+
+		let bannerType;
+
+		if ( nextAppointment ) {
+			bannerType = 'upcomingAppointment';
+		} else if ( hasAvailableConciergeSessions ) {
+			if ( 1 === scheduleId ) {
+				bannerType = 'availableIncludedSession';
+			} else if ( 2 === scheduleId ) {
+				bannerType = 'availablePurchasedSession';
+			}
+		} else {
+			bannerType = 'canPurchaseConcierge';
```

Comment:
All the banner type strings, e.g. `'upcomingAppointment`, `canPurchaseConcierge`, are used across `purchases-list` and `concierge-banner`. It would be safer if we define them as shared constants somewhere.

The other possible way is to pass the relevant props to `ConciergeBanner` from here so these constants are contained inside of it, hence no need of sharing them. Of course, it's up to you to decide as long as we de-dupe them :)

Reply:
Fixed in [4d6a1d9](https://github.com/Automattic/wp-calypso/pull/34956/commits/4d6a1d996874a6dff8f16889b1bf560f27a168fe) and [7379361](https://github.com/Automattic/wp-calypso/pull/34956/commits/73793615cc0772b5e6c2c9d6712e3fbea946cf0d)